packages/Utils/src/DTK_Core.cpp

```diff
@@ -39,7 +37,7 @@ void initKokkos( int *argc, char ***argv )
         if ( !kokkosIsInitialized )
         {
             // Unlike MPI_Init, Kokkos promises not to modify argc and argv.
-            Kokkos::initialize( *argc, *argv );
+            Kokkos::initialize( std::forward<Args &>( args )... );
```

Comment:
..., and you added a `&` here.

Would you please comment that we could not use perfect forwarding and explain what is happening there?
Or did you try each combination until the compiler stopped complaining?